### PR TITLE
Adding some integration tests

### PR DESCRIPTION
**Pull request by mieciu**
These should really boost our confidence when making changes, esp. w.r.t routing and configuration (but not only of course 😉) 

Full suite run on GitHub CI lasts <4 minutes. Locally <2 minutes given all the images are available.

### CODE REVIEW

File: ci/it/testcases/test_reading_clickhouse_tables.go [hunk 1]
```diff
@@ -63,3 +66,43 @@ func (a *ReadingClickHouseTablesIntegrationTestcase) testRandomThing(ctx context
...
+	if err := json.Unmarshal(bodyBytes, &jsonResponse); err != nil {
+		t.Fatalf("Failed to unmarshal response body: %s", err)
+	}
+	hits, _ := jsonResponse["hits"].(map[string]interface{})
+	// We should get proper search result from Elasticsearch
+	hit := hits["total"]
+	hitValue := hit.(map[string]interface{})["value"]
+	assert.Equal(t, float64(1), hitValue)
+	assert.Contains(t, string(bodyBytes), "Alice")
+	assert.Equal(t, 200, resp.StatusCode)
```
Comment (avelanarius) [inline ci/it/testcases/test_reading_clickhouse_tables.go:102]: https://github.com/QuesmaOrg/quesma/commit/7aca4d05397f84c0fdc8ca7bb90d54b8d55af424 😉

Comment (mieciu) [reply ci/it/testcases/test_reading_clickhouse_tables.go]: HAHAHA GOTCHA!!

**Comment by mieciu** [comment]
Complete run https://github.com/QuesmaOrg/quesma/actions/runs/11293285045

File: ci/it/testcases/test_dual_write_and_common_table.go [hunk 2]
```diff
@@ -0,0 +1,335 @@
+package testcases
```
Comment (jakozaur) [inline ci/it/testcases/test_dual_write_and_common_table.go:1]: Add license header.

**Review by jakozaur — APPROVED** [review]

**Pull request merged**